tests/transformer/test_try.py

```diff
@@ -47,6 +48,31 @@ def test_RestrictingNodeTransformer__visit_Try__2(
     ])
 
 
+TRY_EXCEPT_STAR = """
+def try_except_star(m):
+    try:
+        m('try')
+        raise ExceptionGroup("group", [IndentationError('f1'), ValueError(65)])
+    except* IndentationError:
+        m('IndentetionError')
+    except* ValueError:
+        m('ValueError')
+"""
+
+
+def test_RestrictingNodeTransformer__visit_Try__3(mocker):
+    """It allows try-except statements."""
+    if IS_PY311_OR_GREATER:
```

Comment:
That check should be replaced with the pytest marker annotation

Reply:
Fixed